charts/argocd-notifications/values.yaml

```diff
@@ -35,6 +35,17 @@ secret:
       token:
       # Optional override username
       username:
+      # Optional override icon
+      icon:
+      # Optional override signingSecret: https://argoproj-labs.github.io/argocd-notifications/recipients/slack-bot/
+      signingSecret:
```

Comment:
I wonder whether Slack bot configuration should be separate of the notifiers since it is somewhat tangential.  Also I think the [resources needed for Slack bot installation](https://argoproj-labs.github.io/argocd-notifications/recipients/bot/) need to be included in `templates`.  Thoughts?

Reply:
I think is better to have same structure as in https://github.com/argoproj-labs/argocd-notifications/blob/582ae88639207d38c704a43390384201a5ed8630/notifiers/slack.go#L23 code and not separate configuration.
I adapt slack bot installation manifests to helm chart on days

Reply:
Oh, i got... Slack *BOT* configurations, yes, sure. I think it should be in root configuration node:

```yaml
slack_bot/slack:bot:/slack:/...:
  enabled: false
  ...
```